Companions/Rococo/RococoDialogues.cs

```diff
@@ -14,7 +14,7 @@ public override string GreetMessages(Companion companion)
             switch (Main.rand.Next(4))
             {
                 case 0:
-                    return "\"At first, the creature got surprised after seeing me, then starts laughing out of happiness.\"";
+                    return "\"At first, the creature got surprised after seeing me, then starts laughing out do to happiness.\"";
```

Comment:
???

Reply:
Uh, now I don't know what I did there at all lol. Let me fix that

Reply:
Fixed